### DISCUSSION TITLE
format support: *.csp from TheFoundry Nuke MatchGrade

### DISCUSSION THREAD

**Discussion by remisdemis** [videovillage/community#26] [Ideas]
Hey all,

is it possible to add support for *.csp files exported from the MatchGrade node from Nuke?

mikrosimage's lut_to_lut tool supports claims to support csp files (not validated) so it should be possible.

If needed I could provide a set of files.

Thanks for any Feedback!
Cheers,
Timon

**Comment by gregcotten** [comment]
Hi Timon,

Certainly! Send a dropbox link with a .csp file so I can verify the LUT
format. Make sure to provide any variants on the LUT that may exist (bit
depth, etc.)

Thanks!
Greg

**Comment by scottwillman** [comment]
I had a .csp file that it couldn't read also. Not sure of it's origin, it was given to me from the DI facility. I've attached it here if it's any help.
[HS_EXRlin_To_LogC_1D.csp.zip](https://github.com/videovillage/Lattice-Issues/files/907951/HS_EXRlin_To_LogC_1D.csp.zip)

**Comment by gregcotten** [comment]
Don't currently support .csp files as I never received any example LUTs!
I'll take a look.

**Comment by radioproektor** [comment]
Photoshop also support this LUT format.
[screen shot 2018-01-03 at 6 43 00 am](https://user-images.githubusercontent.com/35053925/34509323-6567c3c6-f051-11e7-8268-a2a7affdc414.png)
